src/Mapper/ProgrammesDbToDomain/CoreEntityMapper.php

```diff
@@ -279,7 +279,8 @@ private function getEpisodeModel(array $dbProgramme): Episode
             $dbProgramme['releaseDate'],
             $dbProgramme['duration'] ?? null,
             $this->castDateTime($dbProgramme['streamableFrom']),
-            $this->castDateTime($dbProgramme['streamableUntil'])
+            $this->castDateTime($dbProgramme['streamableUntil']),
+            $dbProgramme['downloadableMediaSets'] ?? []
```

Comment:
Do you need to do an explode() here or something? I think you'll just get a comma separated string otherwise.

Reply:
Doctrine takes care of the exploding, I tested that this works.

Reply:
huh, fair enough